### PR TITLE
made cairo depend on conf-autoconf

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by samoht** [comment]
Can you add a `.git` at the end of the `dev-repo` url? opam 1.2.0 had some issues.

```
[ERROR] At ~/.opam/repo/default/packages/cairo/cairo.0.4.1/opam:8:49:
  Not a recognised version-control URL
[ERROR] At ~/.opam/repo/default/packages/cairo/cairo.0.4.2/opam:8:49:
  Not a recognised version-control URL
[ERROR] At ~/.opam/repo/default/packages/planets/planets.0.1.14/opam:6:48:
  Not a recognised version-control URL
```

**Comment by UnixJunkie** [comment]
I don't see Travis starting on this pull request. But maybe I'm not enough woken up yet ...

**Comment by samoht** [comment]
I cannot merge:

> This branch has conflicts that must be resolved

**Comment by UnixJunkie** [comment]
It should be OK now

File: packages/planets/planets.0.1.14/opam [hunk 1]
```diff
@@ -3,7 +3,7 @@ author: "Yaron M. Minsky"
 homepage: "https://bitbucket.org/yminsky/planets"
 bug-reports: "yminsky@gmail.com"
 maintainer: "yminsky@gmail.com"
-#dev-repo: "https://bitbucket.org/yminsky/planets"
+dev-repo: "https://bitbucket.org/yminsky/planets.git"
```
Comment (yallop) [inline packages/planets/planets.0.1.14/opam]: This isn't the right URL.  Could you please remove this change from the PR?

**Pull request merged**